CMakeLists.txt

```diff
@@ -248,7 +248,8 @@ set(CORE_SOURCE
         src/backend/MeshRecordComponent.cpp
         src/backend/PatchRecord.cpp
         src/backend/PatchRecordComponent.cpp
-        src/backend/Writable.cpp)
+        src/backend/Writable.cpp
+        src/benchmark/mpi/OneDimensionalBlockSlicer.cpp)
```

Comment:
Is it possible that this file and `mpi_benchmark.cpp` should be just in an extra or example target list?

Reply:
I added `mpi_benchmark.cpp` (now `8_mpi_benchmark.cpp`) to the example target list.
For `OneDimensionalBlockSlicer.cpp` do you mean to create a new list for that?

Reply:
Yes, but this `.cpp` file is not part of the libraries `CORE_SOURCE` but of the examples/apps only.